tests/images/debian-systemd/files/tedge.toml

```diff
@@ -1,3 +1,4 @@
 
 [apt]
 name = "(tedge|c8y|python|wget|vim|curl|apt|mosquitto|ssh|sudo|rolldice).*"
+maintainer = ".*(thin-edge.io).*"
```

Comment:
> Parent suite setup failed: dpkg -i packages/tedge_*.deb returned an unexpected exit code
TOML parse error at line 4, column 34

The system test is failing because a trailing new line is missing.

Reply:
That's unexpected, but good to have a line ending anway...